src/styles/order-service.less

```diff
@@ -21,24 +23,32 @@
   margin-top: 20px;
 
   .order-service-deails-top {
-    height: 70%;
-    .service-title {
-      display: flex;
-      align-items: center;
-      .icon {
+    .icon {
```

Comment:
i noticed this earlier but didnt want to touch it, there is a typo in class name order-service-details-top